### ISSUE TITLE
Scale on printed maps

### ISSUE THREAD

**Issue by miszokienka** [uprel/gisapp#93]
Hello!
Recently I realised that if I have gisapp project set to EPSG:3857 to use Google Maps API Layers then when I measure an object length in gisapp window with measurement utility then I get real value, the same as in the terrain. But when I make a print in scale then after measuring the same object on the sheet of paper and multiplying by the scale it has improper values, i.e line that has 164m in terrain and in gisapp window has 274m on the printed document.
One of solutions is to make qgs project use geodetic EPSG (for this area it's EPSG:2178) but then I will not be able to use Google Maps background anymore.
Is there any other solution?

**Comment by uprel** [comment]
That's a big difference, didn't know can be that much. You were using numeric scale? One reason for difference can be if you generate A4 PDF file you need to be careful that you print it in 100% without any scaling. Can be that print will be adjusted to printer margins and so on, so you must be careful there. You can even generate A3 document and print it on A4 for example, so in that case numeric scale will be way off.

**Comment by miszokienka** [comment]
Yes, only numeric scale is bad. Scale on a scale bar is working good. I'm very careful with that scale settings, I'm printing A4 layout on A4 paper and with disabled 'fit to paper size' option.
Maybe this is a QGIS Server 2 bug? I'll try to migrate to QGIS Server 3 in a few days/weeks.
